docroot/sites/grad.uiowa.edu/modules/grad_core/templates/node--mentor--teaser.html.twig

```diff
@@ -71,85 +71,43 @@
  */
 #}
 
-{#
-/**
- * @file
- * Default theme implementation to display a view of unformatted rows.
- *
- * Available variables:
- * - title: The title of this group of rows. May be empty.
- * - rows: A list of the view's row items.
- *   - attributes: The row's HTML attributes.
- *   - content: The row's content.
- * - view: The view object.
- * - default_row_class: A flag indicating whether default classes should be
- *   used on rows.
- *
- * @see template_preprocess_views_view_unformatted()
- *
- * @ingroup themeable
- */
-#}
-    {{ attach_library('uids_base/person') }}
-    {%
-      set classes = [
-      'card--list',
-      'card--person-teaser',
-      'bg--white',
-      'card--media-left',
-      'media--border',
-      'card--centered-left',
-      'card--alignment-left'
-    ]
-    %}
+{%
+  set classes = [
+  'card--list',
+  'card--person-teaser',
+  'bg--white',
+  'card--media-left',
+  'media--border',
+  'card--centered-left',
+  'card--alignment-left',
+  'card',
+]
+%}
 
+{{ attach_library('classy/node') }}
 
-<h3>{{ title }}</h3>
-
-{% set person_card_view = {
+{% set mentor_card = {
   'attributes': attributes.addClass(classes),
-  'headline_class': 'card__title headline',
+  'card_text': content|without('title', 'field_image', 'field_person_position', 'field_person_email', 'field_person_phone', 'links'),
+  'card_subtitle': content.field_person_position.0,
```

Comment:
@joewhitsitt Do we need to change this back to `content.field_person_position` to allow for multiple titles? I had forgotten about that part when I made my template comment in the issue.

Reply:
changing back to `content.field_person_position` strings them together on the same line. do you know of an example where we pass a multiple value field to a card component?

Reply:
Should this be resolved in the field template for `field_person_position`?

Reply:
I think that's a good idea.

Reply:
comma separated for the teaser